player/loadfile.c

```diff
@@ -300,19 +290,21 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
         }
     } else if (t->type == STREAM_AUDIO) {
         if (s && s->codec->channels.num)
-            APPEND(b, " %d ch", s->codec->channels.num);
-        if (s && s->codec->samplerate) {
-            char *samplerate = mp_format_double(NULL, s->codec->samplerate / 1000.0,
-                                                4, false, false, true);
-            APPEND(b, " %s kHz", samplerate);
-            talloc_free(samplerate);
-        }
+            APPEND(b, " %dch", s->codec->channels.num);
+        if (s && s->codec->samplerate)
+            APPEND(b, " %d Hz", s->codec->samplerate);
     }
     APPEND(b, ")");
     if (s && s->hls_bitrate > 0)
         APPEND(b, " (%d kbps)", (s->hls_bitrate + 500) / 1000);
     if (t->is_external)
         APPEND(b, " (external)");
+    if (t->default_track)
+        APPEND(b, " (*)");
+    if (t->forced_track)
+        APPEND(b, " (f)");
+    if (t->attached_picture)
+        APPEND(b, " [P]");
```

Comment:
Also instead of hieroglyphs, we could print it in full. Bit longer, but who know what `[P]` means? 

`(default, external, picture)` for example, in most cases there are 1-2 flags per track.

Reply:
I think short form is fine for `(*)` because it's so common

Reply:
It is incomprehensible without reading the code though

Reply:
We can simply document it in the manual

Reply:
`default` is not that long and there is plenty of space (excluding title, which may not fir anyway). I mean sure, we can make users read code/docs to see what (*) means, but how to you even search for this? And why explain output when we can make it self-explainable? No other media player does some cryptic flags.

Reply:
I agree with kasper, I don't see where you could even put in the docs.